docs/JLBP-0015.md

```diff
@@ -23,10 +23,14 @@ in their own `<dependencyManagement>` section and omit the
 versions from the specific modules they import. The versions will
 be set by the BOM. This provides consistent versions for artifacts released
 by a single project.
-Your project's BOM should include every non-test-scoped module in the project.
+Your project's BOM shall include only those non-test-scoped modules in *your* project.
```

Comment:
We haven't been using standards language like SHALL and MUST here. I think this sentence already says this, but we can clarify as you suggest below.